### DISCUSSION TITLE
Tabby Full Screen in Quake Mode, Want Half Screen

### DISCUSSION THREAD

**Discussion by darrylmorley** [Eugeny/tabby#8856] [Q&A]
Hi,

I'm just setting up a new system, I've used Tabby as my primary terminal for a while now and usually it's half the screen height in docked/quake mode, for some reason it is now full screen and no settings appear to remedy that.

Can anyone advise how to make Tabby take up half the screen height, please?

v1.0.197

**Comment by Eugeny** [comment]
This is controlled by the Window -> Docking -> Terminal size setting, what is it set to?

**Reply by darrylmorley** [reply]
"Docked terminal size"? I can set the slider to anything and it has no effect.

**Reply by Eugeny** [reply]
Please capture DevTools console logs at the time of the issue: https://github.com/Eugeny/terminus/wiki/Debug-logs

**Comment by darrylmorley** [comment]
I'm going to close this as I believe that it may have been an issue with my Windows installation. I have since gone back to Windows 10 and no longer have the issue.
